src/Attributes/Help.php

```diff
@@ -28,6 +28,6 @@ public static function handle(\ReflectionAttribute $attribute, CommandInfo $comm
         $args = $attribute->getArguments();
         $commandInfo->setDescription($args['description']);
         $commandInfo->setHelp(@$args['synopsis']);
-        $commandInfo->setHidden(@$args['hidden']);
```

Comment:
Don't have time to look at the code rn but it seems like if setHidden is a thing, then it should do the right thing e.g. call through to addAnnotation($hidden)?